src/main/java/seedu/address/logic/commands/AddScoreCommand.java

```diff
@@ -35,14 +35,14 @@ public class AddScoreCommand extends Command {
             + PREFIX_SCORE_DATE + "2012-08-09 ";
 
     public static final String MESSAGE_SUCCESS = "Added score to Student %1$s: %2$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This score already exists in this student's score list";
+    public static final String MESSAGE_DUPLICATE_SCORE = "This score already exists in this student's score list";
```

Comment:
Maybe can consider to have a more specific error message since quite a number of the tester are confused about this part. Change to "Every score must have different date".

Reply:
Sure, thanks